src/iris/ui/static/js/iris.js

```diff
@@ -2382,24 +2399,21 @@ iris = {
             //fetch apps that have custom categories defined
             $.ajax({
                 type: "GET",
-                url: '/v0/categories/',
-                success: function(resp)
-                {
-                    self.data.$addAppOverrideSelect.empty();
-                    self.data.$addAppOverrideSelect.append($('<option value="">').text('Add Application'));
-
-                    appOverrideCategories = resp;
-                    appOverrideCategories.forEach((appSetting, i) => {
-                        customOverrideApps.add(appSetting['application']);
-                    });
-
-                    Array.from(customOverrideApps).sort().forEach(function(app) {
-                        self.data.$addAppOverrideSelect.append($('<option>').text(app));
-                    });
-                }
-            }).fail(function(){
+                url: '/v0/categories/'
+            }).done(function (resp) {
+                  self.data.$addAppOverrideSelect.empty();
+                  self.data.$addAppOverrideSelect.append($('<option value="">').text('Add Application'));
+                  appOverrideCategories = resp;
```

Comment:
`appOverrideCategories` variable definition is missing the `var` prefix, so this variable becomes a global